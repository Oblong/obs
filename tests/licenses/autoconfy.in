Format: http://dep.debian.net/deps/dep5
Upstream-Name: autoconfy

Files: *
Copyright: Max Mustermann
License: MPL-2.0
 On Debian systems you can find the full text of the MPL 2.0 in 
 /usr/share/common-licenses/MPL-2.0

Files: m4/ax_cxx_compile_stdcxx*m4
Copyright: Max Mustermann
License: other

Files: m4/ax_gcc_func_attribute.m4
Copyright: Max Mustermann
License: GPL

Files:
 m4/fred.m4
 src/fred.c
Copyright: Max Mustermann
License: LGPL
