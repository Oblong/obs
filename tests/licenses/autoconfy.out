Format: http://dep.debian.net/deps/dep5
Upstream-Name: autoconfy

Files: *
Copyright: Max Mustermann
License: MPL-2.0
 On Debian systems you can find the full text of the MPL 2.0 in 
 /usr/share/common-licenses/MPL-2.0

Files:
 m4/fred.m4
 src/fred.c
Copyright: Max Mustermann
License: LGPL
